deepmd/utils/argcheck.py

```diff
@@ -332,14 +332,16 @@ def modifier_variant_type_args():
 
 #  --- model compression configurations: --- #
 def model_compression():
-    doc_compress = "The name of the frozen model file."
+    doc_compress = f"The name of the frozen model file."
     doc_model_file = f"The input model file, which will be compressed by the DeePMD-kit."
     doc_table_config = f"The arguments of model compression, including extrapolate(scale of model extrapolation), stride(uniform stride of tabulation's first and second table), and frequency(frequency of tabulation overflow check)."
+    doc_min_nbor_dist = f"The nearest distance between neighbor atoms saved in the frozen model."
```

Comment:
Why asking users to provide this key?

Reply:
This key is not provided by the users, it is computed by the code

Reply:
I don't think it's a correct way to store `min_nbor_dist`... Here the Argument will also affect the document. Can we cache it in another place?